docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: "3.7"
+
+services:
+  web:
+    image: dhis2/core-dev:latest
```

Comment:
I think this should probably be `local` or some other non-standard tag

Reply:
I'll do it. Its just unfortunate as right now you could do `git clone ... && docker compose up`. This will not be possible anymore. But if we change the other tags to `local` for safety purposes we also need to change this one 👌🏻